doc/subsystems/networking/native_posix_setup.rst

```diff
@@ -0,0 +1,86 @@
+.. _networking_with_native_posix:
+
+Networking with native_posix board
+##################################
+
+This page describes how to set up a virtual network between a (Linux) host
+and a Zephyr application running in a native_posix board.
+
+In this example, the :ref:`echo-server-sample` sample application from
+the Zephyr source distribution is run in native_posix board. The Zephyr
+native_posix board instance is connected to a Linux host using a tuntap device
+which is modeled in Linux as an Ethernet network interface.
+
+Prerequisites
+*************
+
+On the Linux Host, fetch the Zephyr ``net-tools`` project, which is located
+in a separate Git repository:
+
+.. code-block:: console
+
+   git clone https://github.com/zephyrproject-rtos/net-tools
+
+
+Basic Setup
+***********
+
+For the steps below, you will need three terminal windows:
+
+* Terminal #1 is terminal window with net-tools being the current
+  directory (``cd net-tools``)
+* Terminal #2 is your usual Zephyr development terminal,
+  with the Zephyr environment initialized.
+* Terminal #3 is the console to the running Zephyr instance (optional).
```

Comment:
Can we write here explicitly "to the running Zephyr native_posix instance", because to a user who's familiar with e.g. QEMU running, but not native_posix, this phrase sounds strange.